### ISSUE TITLE
Collapsible sidebar Unicode chevron render issue

### ISSUE THREAD

**Issue by CaptainT33mo** [facebook/Docusaurus#1207]
## 🐛 Bug Report

The chevron (dropdown icon) on the Collapsible Sidebar is rendering differently on different systems.

## To Reproduce

I am running [docursaurus](https://docusaurus.io/docs/en/installation) on Chrome 71 for Windows 10 version 1809.

## Expected behavior

This is how it should render in all systems [https://i.postimg.cc/T1S3qD9h/Screen-Shot-2019-01-28-at-10-57-10-AM.png](https://i.postimg.cc/T1S3qD9h/Screen-Shot-2019-01-28-at-10-57-10-AM.png).

## Actual Behavior

This is how it is rendering in my system [https://i.postimg.cc/G3MWgr1P/86775128-61-E9-474-B-8-CEA-E01-DDA855-B1-F-png.jpg](https://i.postimg.cc/G3MWgr1P/86775128-61-E9-474-B-8-CEA-E01-DDA855-B1-F-png.jpg)

## Reproducible Demo

Open [docursaurus](https://docusaurus.io/docs/en/installation) on Chrome 71 for Windows 10 version 1809.

## Suggestion

To make it consistent across all devices, this Unicode chevron can be replaced with **SVG**.

@yangshun

**Comment by CaptainT33mo** [comment]
> ## 🐛 Bug Report
> The chevron (dropdown icon) on the Collapsible Sidebar is rendering differently on different systems.
> 
> ## To Reproduce
> I am running [docursaurus](https://docusaurus.io/docs/en/installation) on Chrome 71 for Windows 10 version 1809.
> 
> ## Expected behavior
> This is how it should render in all systems https://i.postimg.cc/T1S3qD9h/Screen-Shot-2019-01-28-at-10-57-10-AM.png.
> 
> ## Actual Behavior
> This is how it is rendering in my system https://i.postimg.cc/G3MWgr1P/86775128-61-E9-474-B-8-CEA-E01-DDA855-B1-F-png.jpg
> 
> ## Reproducible Demo
> Open [docursaurus](https://docusaurus.io/docs/en/installation) on Chrome 71 for Windows 10 version 1809.
> 
> ## Suggestion
> To make it consistent across all devices, this Unicode chevron can be replaced with **SVG**.

@yangshun

**Comment by yangshun** [comment]
I'll be fixing it with the SVG provided by @CaptainT33mo: 

```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="#565656" d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z"></path><path d="M0 0h24v24H0z" fill="none"></path></svg>
```

via inline SVG or CSS images and remove the unicode.

**Comment by yangshun** [comment]
Fixed and released in v1.7.2
